frontend/pages/hosts/details/cards/Activity/_styles.scss

```diff
@@ -22,6 +22,6 @@
     font-size: $xx-small;
     position: absolute;
     top: 14px;
-    right: 0;
+    right: 14px;
```

Comment:
Interesting, it seems to have assumed this should be the same as the `top` value

Reply:
Definitely an improvement! 😆

I just double-checked the [original designs](https://www.figma.com/design/HpQ9W7z0YrXyznyjQ209Fg/-22866-Upcoming-activities-run-as-listed--one-queue-?node-id=5364-2582&t=6Knp82K7d9qE2WCx-1), and I think we actually need a parent `<div>` to have `position: relative`:
![Screenshot 2025-05-16 at 6 04 24 PM](https://github.com/user-attachments/assets/e6022ab4-5ef6-4347-9440-1bc6c8e900af)